playground/style.css

```diff
@@ -1442,13 +1441,42 @@
   color: #003b5c;
 }
 
+
 .rxntRowStyle{
   font-weight: normal;
   background-color: #ffffff;
-  border: solid 1px #dadada;
+  border: solid 1px #dadada !important;
```

Comment:
Are these !important absolutely necessary?

Reply:
yes @slishak  , it was not working without important, we have tried , I was taking some inline css from schema-generator